libraries/botbuilder-core/botbuilder/core/activity_handler.py

```diff
@@ -344,3 +351,56 @@ class ActivityHandler:
             conversation update, message reaction, or event activity, it calls this method.
         """
         return
+
+    async def on_invoke_activity(  # pylint: disable=unused-argument
+        self, turn_context: TurnContext
+    ):
```

Comment:
john is makign some changes to invoke to make it more consistent with teams. you might want to port that work to python or have him port it (he is doing it for c# and js).
i am not sure what the work is

Reply:
see other comments.
do we need to refer to HTTPStatus and StatusCodes?